pkg/cmd/cli/describe/describer.go

```diff
@@ -283,17 +283,16 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 		if len(builds.Items) == 0 {
 			return nil
 		}
-		fmt.Fprintf(out, "Builds(Name,Status,Creation Time)\n")
+		fmt.Fprintf(out, "Builds(Name,Status,Duration,Creation Time)\n")
 		sortedBuilds := sortableBuilds(builds.Items)
 		sort.Sort(sortedBuilds)
-		c := 0
 		for i := range sortedBuilds {
 			// iterate backwards so we're printing the newest items first
 			build := sortedBuilds[len(sortedBuilds)-1-i]
-			fmt.Fprintf(out, fmt.Sprintf("- %s %s %v\n", build.Name, build.Status, build.CreationTimestamp.Rfc3339Copy().Time))
-			c++
+			duration := build.CompletionTimestamp.Rfc3339Copy().Time.Sub(build.StartTimestamp.Rfc3339Copy().Time)
```

Comment:
Yeah, I was about to write that, after carefully studying the code you've pointed to, where I see you also print waiting time as well.